.github/workflows/license-checker.yml

```diff
@@ -14,7 +32,7 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-
-      - name: run license check
-        run: |
-          make license-check
+      - name: Check License Header
+        uses: apache/skywalking-eyes@v0.1.0
```

Comment:
Maybe it is more appropriate to use `apache/skywalking-eyes@main` here

Reply:
fixed by #510